pkg/cmd/attestation/verify/verify.go

```diff
@@ -166,7 +166,7 @@ func runVerify(opts *Options) error {
 		if ok := errors.Is(err, api.ErrNoAttestations{}); ok {
 			return fmt.Errorf("no attestations found for subject: %s", artifact.DigestWithAlg())
 		}
-		return fmt.Errorf("failed to fetch attestations for subject: %s", artifact.DigestWithAlg())
+		return fmt.Errorf("failed to fetch attestations for subject: %s: %v", artifact.DigestWithAlg(), err)
```

Comment:
Regardless of whether we drop the the file extension requirement or not, we should definitely keep this part!